docs/src/pages/guides/notifications.md

```diff
@@ -1,3 +1,189 @@
-# Notifications
+---
+id: notifications
+title: Notifications
+---
 
-under construction
+## Notification Types
+
+In Monika, currently we provided multiple notifications type. We are implementing email notifications through sendgrid, mailgun, smtp and webhook.
```

Comment:
~In Monika, currently we provided multiple notifications type.~ Monika provides multiple notification types.

~We are implementing email notifications through sendgrid, mailgun, smtp and webhook.~ You can implement notifications via sendgrid, mailgun, smtp and/or webhook.

Reply:
modified